src/app/pages/common/entity/entity-table/entity-table.component.html

```diff
@@ -245,6 +268,10 @@
             <div class="clickable"
               *ngIf="col.toggle"
               id="{{row.name}}_{{col.name}}-overlay"
+              [ix-auto]="col?.name"
+              ix-auto-type="clickable-overlay"
```

Comment:
Shortening this seems fine to me:
```suggestion
              ix-auto-type="overlay"
```

Reply:
done!